src/Plugin.js

```diff
@@ -28,12 +28,12 @@ module.exports = class extends ChainedMap {
       this.set('args', obj.args);
     }
 
-    return this;
+    return super.merge(obj, [...omit, 'args', 'plugin'])
```

Comment:
Missing semicolon.

I wonder if we should add some linting to this repo?

Reply:
Yeah, I've put it off for a while. I can do that in another PR, although it would be easier to either use Neutrino (seems circular) or move this repo into the monorepo.